OpenTabletDriver.Plugin/CoreComponents/ICompositeDeviceHub.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+
+namespace OpenTabletDriver.Plugin.Devices
```

Comment:
Namespace must be updated, I also want to suggest a different name as well.
```suggestion
namespace OpenTabletDriver.Plugin.Components
```